integrationTests/chainSimulator/bridge/esdtSafe_test.go

```diff
@@ -0,0 +1,529 @@
+package bridge
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/big"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/stretchr/testify/require"
+
+	"github.com/multiversx/mx-chain-go/config"
```

Comment:
go mod import

Reply:
This should be correct. There are 3 categories:
1. go imports
2. external github imports
3. internal imports